### ISSUE TITLE
Suggest EndorsementRating subtype of Rating to indicate supportive ratings

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1293]
Currently we have "Rating: A rating is an evaluation on a numeric scale, such as 1 to 5 stars."

Suggest amending to "an evaluation, commonly on a numeric scale", and creating a subtype EndorsementRating with definition "A positive rating that expresses a positive evaluation or endorsement."

Justification: some situations e.g. "pick of the week" critic's reviews, are implicitly positive. However if a numeric scale is not used, it is impossible to tell that the Rating should be taken as an endorsement.

We should also note that Rating is associated with some properties e.g. bestRating that assume a numeric scale ("The highest value allowed in this rating system. If bestRating is omitted, 5 is assumed."). We may need to soften this language to deal with non-numeric, non-scaled endorsement ratings (if necessary an unscaled endorsement can be seen as implicitly 5/5).

A related option would be for a Text-values of http://schema.org/ratingValue (e.g. "Critic's pick") to indicate a non-scalar system, in which case bestRating and worstRating would say "if bestRating is omitted and ratingValue is a number, 5 can be assumed."

Nearby: #780 for starRating discussion

/cc @gmackenz

**Comment by danbri** [comment]
The sketch came less from thinking about +1/Like/etc in the social network / media sense, more like review+rating in a newspaper or blog. But perhaps they're pretty much the same.

**Comment by danbri** [comment]
I've had a go at implementing this in pending, and in doing so articulating how this relates to EndorseAction.

-> http://pending.webschemas.org/EndorsementRating

If we go for this, let's also update and link from http://schema.org/EndorseAction too.

How does this look?

**Comment by danbri** [comment]
This is seeing some usage, e.g. Zagat, Fodors. 

Let's move it along into core.

```
... 
{"@type":"EndorsementRating","name":"Zagat Rated","ratingValue":"4.5"}]}}
```

(seems to be Javascript-injected, in https://www.zagat.com/r/smoke-shack-bbq-san-antonio if you look via browser inspection)

Also https://www.fodors.com/world/north-america/usa/illinois/chicago/hotels/reviews/hotel-palomar-chicago-477853 -

```

    {
        "@context": "http://www.schema.org",
        "@type": "Hotel",
        "name": "Hotel Palomar Chicago",
        "url": "https://www.fodors.com/world/north-america/usa/illinois/chicago/hotels/reviews/hotel-palomar-chicago-477853",
        "logo": "http://www.fodors.com/images/main-nav-home.svg",
        "image": "https://cimg2.ibsrv.net/cimg/www.fodors.com/500x500_60/66/5a66467f75dee-310066.jpg",
        "description": "Fodor's expert travel writers review the Hotel Palomar Chicago in  and give you the straight scoop, complete with photos, details on the rooms and accommodations, up-to-the-minute pricing via our Rate Checker, local restaurant and bar recommendations, and insider tips for maximizing your stay",
        "telephone": "312-755–9703;877-731–0505",
        "address": {
            "@type": "PostalAddress",
            "streetAddress": "505 N. State, Chicago, Illinois, 60654, USA",
            "addressLocality": "",
            "addressRegion": "",
            "postalCode": "",
            "addressCountry": "",
            "telephone": "312-755–9703;877-731–0505"
        },
        "review": [{
            "@type": "Review",
            "author": "Kristine Hansen",
            "description": "Fodor's expert travel writers review the Hotel Palomar Chicago in  and give you the straight scoop, complete with photos, details on the rooms and accommodations, up-to-the-minute pricing via our Rate Checker, local restaurant and bar recommendations, and insider tips for maximizing your stay",
            "name": "Hotel Palomar Chicago",
            "reviewRating": [{
                "@type": "Rating",
                "bestRating": "5",
                "ratingValue": "4",
                "worstRating": "1"
            },{
                "@type": "EndorsementRating",
                "name": "4 thumbs up",
                "ratingValue": "4"
            }]
        }]
    }
   ```

We are using this at Google, e.g. see https://developers.google.com/search/docs/data-types/critic-review.

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
